docker/stubs/flac/Dockerfile

```diff
@@ -0,0 +1,21 @@
+FROM ouspg/libfuzzer-base
+
+MAINTAINER https://github.com/ouspg/libfuzzerification
+ENV TARGET 'flac'
+
+#Install dependencies and fetch the source.
+
+RUN apt-get update && apt-get upgrade -y && apt-get install -y libflac-dev pkg-config libtool-bin && apt-get build-dep libflac-dev -y
+RUN cd /src && git clone https://git.xiph.org/flac.git
+ADD flac_fuzzer.c /src/flac/
+
+ADD ${TARGET}_fuzzer.c /src/$TARGET/
+RUN mkdir -p /work/$TARGET/
+
+#Build
+ADD build.sh /src/scripts/
+RUN bash /src/scripts/build.sh
```

Comment:
Maybe copy `build.sh` script directly under `/src/flac/`? It's job is to build flac so it would be more obvious place. Then set `WORKDIR /src/flac` and `RUN ./build.sh`. That eliminates the `cd /src/flac` from `build.sh`.